ARCH-882 - Action tweaks (#2)
im-open/process-jest-test-results: README.md

This action does not run the Jest tests itself and it can only process one results file at a time.

- [process-jest-test-results](#process-jest-test-results)
  * [Failures](#failures)
  * [Limitations](#limitations)
  * [Action Outputs](#action-outputs)
  * [Inputs](#inputs)
  * [Outputs](#outputs)
  * [Usage Examples](#usage-examples)
  * [Recompiling](#recompiling)
  * [Code of Conduct](#code-of-conduct)
  * [License](#license)
  
## Failures
The status check can be seen as a new item on the workflow run, a PR comment or on the PR Status Check section.  If the test results contain failures, the status check will be marked as failed. Having the status check marked as failed will prevent PRs from being merged. If this status check behavior is not desired, the `ignore-test-failures` input can be set and the outcome will be marked as neutral if test failures are detected. The status badge that is shown in the comment or status check body will still indicate it was a failure though.

## Limitations

> GitHub does have a size limitation of 65535 characters for a Status Check body or a PR Comment.  This action will fail if the test results exceed the GitHub [limit].  To mitigate this size issue only failed tests are included in the output.

Reviewer: ```suggestion
GitHub has a size limitation of 65535 characters for a Status Check body or a PR Comment.  This action will fail if the test results exceed the [limit].  To mitigate this size issue, only failed tests are included in the output.
```
